src/main/scala/cromwell/webservice/CromwellApiHandler.scala

```diff
@@ -108,6 +112,16 @@ class CromwellApiHandler(workflowManager: ActorRef) extends Actor {
         case Failure(ex: WorkflowManagerActor.CallNotFoundException) => context.parent ! RequestComplete(StatusCodes.NotFound, s"Call $callFqn not found for workflow '$id'.")
         case Failure(ex) => context.parent ! RequestComplete(StatusCodes.InternalServerError, ex.getMessage)
       }
+
+    case CallStdoutStderr(id, callFqn) =>
+      val eventualCallLogs = ask(workflowManager, WorkflowManagerActor.CallStdoutStderr(id, callFqn)).mapTo[(WdlFile, WdlFile)]
+      eventualCallLogs onComplete {
+        case Success(logs) => context.parent ! RequestComplete(StatusCodes.OK, CallStdoutStderrResponse(id.toString, callFqn, logs._1.value, logs._2.value))
+        case Failure(ex: WorkflowManagerActor.WorkflowNotFoundException) => context.parent ! RequestComplete(StatusCodes.NotFound, ex.getMessage)
```

Comment:
Sure but the lhs might be more trouble than worth. At the very least the rhs should be pulled into a function